packages/ckeditor5-ui/src/formheader/formheaderview.ts

```diff
@@ -61,41 +66,69 @@ export default class FormHeaderView extends View {
 	 */
 	constructor(
 		locale: Locale | undefined,
-		options: {
-			label?: string | null;
-			class?: string | null;
-			icon?: string | null;
-		} = {}
+		options: FormHeaderViewOptions = {}
 	) {
 		super( locale );
 
 		const bind = this.bindTemplate;
 
+		this.options = options;
+		this.iconView = this._createIcon();
+
 		this.set( 'label', options.label || '' );
 		this.set( 'class', options.class || null );
 
 		this.children = this.createCollection();
 
+		if ( this.options.left ) {
+			this.children.add( this.options.left );
+		}
```

Comment:
What does `left` mean? Maybe we could name it more precisely? Same for `right`

Reply:
Renamed to `leftSlot` and `rightSlot`.